build-tools/scripts/y2metainfo

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+#
+# y2metainfo - Automagically generate appstream data for YaST modules based on their desktop files
+#
+# Author: Stasiek Michalski <hellcp@opensuse.org>
+# Copyright 2019
+#
+
+require "rexml/document"
+require "optparse"
+require "inifile"
+
+options = {}
+
+OptionParser.new do |parser|
+  parser.banner = "Usage: y2metainfo [options]"
+  parser.on("-h", "--help", "Show this help message") do
+    puts parser
```

Comment:
does it work? I expect here `parser.help` or `parser.to_s` ( its alias ).

Reply:
```sh
lcp@world:~/Documents/git/yast-devtools/build-tools/scripts> ./y2metainfo --help
Usage: y2metainfo [options]
    -h, --help                       Show this help message
    -f, --file FILEPATH              The desktop file of the module
    -l, --license LICENSE            The license of the module
    -o, --output DIRECTORY           The output directory
```

Although I get an error for a different reason after that, I should fix that